src/rafterydiag.jl

```diff
@@ -77,15 +78,22 @@ function rafterydiag(
         )
     end
 
-    colnames = tuple(Symbol.(["parameters", "Thinning", "Burn-in", "Total", "Nmin",
-        "Dependence Factor"])...)
+    # Retrieve columns.
+    columns = [[vals[k][:, i] for i in 1:5] for k in 1:m]
 
-    pnames = Symbol.(names(c))
-    vals = map(x -> round.(x, digits=4), vals)
-    columns = [vcat([pnames], [vals[k][:,i] for i in 1:5]) for k in 1:m]
+    # Obtain names of parameters.
+    names_of_params = names(chn)
 
-    dfs = [NamedTuple{colnames}(tuple(columns[k]...)) for k in 1:m]
-    dfs_wrapped = [ChainDataFrame("Raftery and Lewis Diagnostic - Chain $k",
-                   dfs[k]) for k in 1:m]
-    return dfs_wrapped
+    # Compute data frames.
+    vector_of_df = [
+        ChainDataFrame(
+            "Raftery and Lewis Diagnostic - Chain $i",
+            (parameters = names_of_params, Thinning = column[1], var"Burn-in" = column[2],
+             Total = column[3], Nmin = column[4], var"Dependence Factor" = column[5]);
+            digits = digits
```

Comment:
```suggestion
            (parameters = names_of_params, Thinning = column[1], burn_in = column[2],
             total = column[3], n_min = column[4], dependence_factor = column[5]);
            digits = digits
```

Lowercase, no `@var_str`. See above.